### DISCUSSION TITLE
Bookmark/Favorite Emote list

### DISCUSSION THREAD

**Discussion by buzz** [Chatterino/chatterino2#4317] [Ideas] [answered]
Make it possible to fav emotes with Ctrl+mouse click or similar. They should appear in a separate emote tab for quick access.

Only mention of this feature I found: #33

**Comment by beastyrabbit** [comment]
Hi, i also really would like to have a Feature like this.
Even if it is not perfect. But better then looking for your emotes every time. 

The only way to quick find at the moment is to remember every emote you want to use. 
and you also not easy able to use emotes that are "in trend".

Regards
Beasty

**Comment by Xanthognarh** [comment]
I like the idea of quickly setting favorite emotes. But I’d also like a more advanced way to organize Twitch emotes (7TV/BTTV/etc. would be more complex) in addition:

- Category list
  - Split the emote window vertically to add a persistent list of categories that remains visible regardless of the selected emote tab.
  - Users can add, rename, and remove categories (e.g., via a context menu on the list).
  - Drag & drop emotes into categories to add or remove them from that category’s collection; on hover/drag of an emote highlight categories where this emote is already in that category.

- Custom/Favorites tab
  - Add a "Custom"/"Favorites" tab to the emote window.
  - Display each category with: category name, rename button, delete button and linebreak.
  - Show every emote assigned to the category and include placeholders for missing emotes (e.g., seasonal or expired subscription - do not auto-delete them).
  - Use separators between categories.

This lets users group emotes into meaningful custom categories (e.g., hi, cheer, bye) so they can quickly find the right emotes. This also allows for a greater variety of emojis to be used.

The implementation is relatively simple: you have one set per category and simply enter the names of the emotes.
To display the emotes in the Favorites tab, data from the other tabs is used. This eliminates the need to convert the text into images via API and also checks whether the emotes are usable.

**Comment by pajlada** [comment, marked as answer]
The idea of favouriting an emote has been approved, and I have created an issue for this here: https://github.com/Chatterino/chatterino2/issues/6931
